index.js

```diff
@@ -1,15 +1,27 @@
-/* jshint node: true */
-'use strict';
 
+'use strict';
+var path = require('path');
 module.exports = {
   name: 'ember-cli-cloudinary',
 
   included: function(app) {
     this._super.included(app);
-
-    app.import(app.bowerDirectory + '/blueimp-file-upload/js/vendor/jquery.ui.widget.js');
-    app.import(app.bowerDirectory + '/blueimp-file-upload/js/jquery.iframe-transport.js');
-    app.import(app.bowerDirectory + '/blueimp-file-upload/js/jquery.fileupload.js');
+    
+    var blueimpFileUploadPath = path.join(app.bowerDirectory,'/blueimp-file-upload/');
+    var blueimpLoadImagePath = path.join(app.bowerDirectory,'/blueimp-load-image/');
+    var blueimpblueimpCanvasToBlobPath = path.join(app.bowerDirectory,'/blueimp-canvas-to-blob/');
+    
+    app.import(path.join(blueimpFileUploadPath, 'js/vendor/jquery.ui.widget.js'));
+    app.import(path.join(blueimpFileUploadPath, 'js/jquery.iframe-transport.js'));
+    app.import(path.join(blueimpFileUploadPath, 'js/jquery.fileupload.js'));
     app.import(app.bowerDirectory + '/cloudinary-jquery-file-upload/cloudinary-jquery-file-upload.js');
+    app.import(path.join(blueimpLoadImagePath,           'js/load-image.all.min.js'));
```

Comment:
Don't attempt to make everything aligned, follow standard Ember convention as the original has it and you will do fine.